### ISSUE TITLE
GeoSwitch Plugin Not Activating

### ISSUE THREAD

**Issue by lacoudhir** [elialgranti/geoswitch#6]
I'm getting this error:
Parse error: syntax error, unexpected T_STRING in /wp-content/plugins/geoswitch/class.geoswitch.php on line 54

**Comment by elialgranti** [comment]
I'll take a look what version of WP are you using?

**Comment by lacoudhir** [comment]
Ok thanks. Using WP 4.3

-- 
lisa eyo  |  lacoudhir

email: lisa@lacoudhir.com
mobile: +61 419 896 113 (australia)
cell: +1778 707 3359 (canada)
skype: lisa.eyo
twitter: @lacoudhirdesign

step out of the ordinary…

From:  Eli Algranti
Reply-To:  elialgranti/geoswitch
Date:  Friday, August 21, 2015 at 4:36 PM
To:  elialgranti/geoswitch
Cc:  Lacoudhir
Subject:  Re: [geoswitch] GeoSwitch Plugin Not Activating (#6)

I'll take a look what version of WP are you using?

**Comment by elialgranti** [comment]
Hi,

Sorry for the delay. I just downloaded the plugin from Wordpress and tried
it in a new installation and everything works as expected.
What version of PHP are you using? I used 5.5.

Regards,
Eli

**Comment by lacoudhir** [comment]
Hi Eli

Sorry for the delay in thanks. I just checked this today with a switchto 5.4 and it seems to work now.
Thanks!

From:  Eli Algranti
Reply-To:  elialgranti/geoswitch
Date:  Sunday, August 23, 2015 at 4:50 PM
To:  elialgranti/geoswitch
Cc:  Lacoudhir
Subject:  Re: [geoswitch] GeoSwitch Plugin Not Activating (#6)

Hi,

Sorry for the delay. I just downloaded the plugin from Wordpress and tried
it in a new installation and everything works as expected.
What version of PHP are you using? I used 5.5.

Regards,
Eli
